### DISCUSSION TITLE
Log

### DISCUSSION THREAD

**Discussion by Yumoox** [YimMenu/YimMenu#3180] [Q&A]
is there a way to Log script events to see what ppl are doing?

**Comment by Rxann** [comment]
Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.

**Reply by Yumoox** [reply]
> Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.

yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events

**Reply by Rxann** [reply]
> > Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.
> 
> yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events

There is under debug, but it should not be used by a regular user. I mean, who am I to tell you how to use it but 99% of it will likely mean nothing to you.

**Reply by Yumoox** [reply]
> > > Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.
> > 
> > 
> > yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events
> 
> There is under debug, but it should not be used by a regular user. I mean, who am I to tell you how to use it but 99% of it will likely mean nothing to you.

do you know how to use it?

**Reply by Rxann** [reply]
> > > > Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.
> 
> > > 
> 
> > > 
> 
> > > yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events
> 
> > 
> 
> > There is under debug, but it should not be used by a regular user. I mean, who am I to tell you how to use it but 99% of it will likely mean nothing to you.
> 
> 
> 
> do you know how to use it?

Go under debug -> logging -> log script events

**Reply by Yumoox** [reply]
> > > > > Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events
> > 
> > 
> > > 
> > 
> > 
> > > There is under debug, but it should not be used by a regular user. I mean, who am I to tell you how to use it but 99% of it will likely mean nothing to you.
> > 
> > 
> > do you know how to use it?
> 
> Go under debug -> logging -> log script events

then what where can i see the script events

**Reply by Rxann** [reply]
> > > > > > Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > There is under debug, but it should not be used by a regular user. I mean, who am I to tell you how to use it but 99% of it will likely mean nothing to you.
> > > 
> > > 
> > > do you know how to use it?
> > 
> > 
> > Go under debug -> logging -> log script events
> 
> then what where can i see the script events

That is not the point of the feature. it gives you the raw hashes because it is meant for developers.

**Reply by Yumoox** [reply]
> > > > > > > Unless you are a developer, this is probably not useful for you. Any important events will prompt a notification that is displayed to you.
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > yea but sometimes i miss it and sometimes they dont come up on the cmd part so i was wondering if there was a spot in the menu where u can see the script events
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > There is under debug, but it should not be used by a regular user. I mean, who am I to tell you how to use it but 99% of it will likely mean nothing to you.
> > > > 
> > > > 
> > > > do you know how to use it?
> > > 
> > > 
> > > Go under debug -> logging -> log script events
> > 
> > 
> > then what where can i see the script events
> 
> That is not the point of the feature. it gives you the raw hashes because it is meant for developers.

ahh i see, nvm then
